api/client/options.proto

```diff
@@ -128,4 +128,6 @@ message CommandLineOptions {
   google.protobuf.BoolValue open_loop = 21;
   // See :option:`--jitter-uniform`
   google.protobuf.Duration jitter_uniform = 25 [(validate.rules).duration.gte.nanos = 1];
+  // See :option:`--backend-endpoint`
+  repeated google.protobuf.StringValue backend_endpoints = 27;
```

Comment:
We talked about this in person, but officially putting it here. Would it be possible to change the API here slightly, so this is in a `oneof` with `uri`, and maybe have a different way of pushing path through?  If that is unduly difficult, feel free to ignore.

Reply:
I updated it so that now the user sets either a URI or these options:
--multi-target-path
one or more --multi-target-endpoint host:port
optionally --multi-target-use-https 

This removes the strangeness of overriding part of the URI, part of the behavior being determined by the URI, etc. In the proto it's clean, and on the CLI it's fully exposed in hopefully a logical way.